isis/src/control/objs/BundleUtilities/BundleObservation.h

```diff
@@ -106,6 +109,8 @@ namespace Isis {
       LinearAlgebra::Vector m_aprioriSigmas;
       //! A posteriori (adjusted) parameter sigmas.
       LinearAlgebra::Vector m_adjustedSigmas;
+      // Test statistic used for determining gross outliers
+      LinearAlgebra::Vector m_grossOutlierTestStatistic;
```

Comment:
This does not seem like a good place to store this. Why not just store it in the measures? If it's stored here we need to keep an index for each measure into the vector to get its test statistic which is cumbersome and error prone.